homeassistant/components/nightscout/sensor.py

```diff
@@ -37,7 +37,7 @@ class NightscoutSensor(Entity):
     def __init__(self, api: NightscoutAPI, name):
         """Initialize the Nightscout sensor."""
         self.api = api
-        self._unique_id = hashlib.sha256(api.server_url.encode("utf-8")).hexdigest()
+        self._unique_id = hash_from_url(api.server_url)
```

Comment:
We can use the config entry unique_id instead. We can access it on the entry as `entry.unique_id`.